assets/css/tables.css

```
@@ -0,0 +1,11 @@
+.costumer-table-data thead {
+  text-align: left;
+}
+
+.costumer-table-data th {
+  padding-right: 15px;
+}
+
+.costumer-table-data td {
```

What about combing these two rules?

```suggestion
.costumer-table-data td,
.costumer-table-data th {
```